static/app/views/starfish/views/appStartup/countChart.tsx

```diff
@@ -0,0 +1,143 @@
+import {getInterval} from 'sentry/components/charts/utils';
+import {t} from 'sentry/locale';
+import type {MultiSeriesEventsStats} from 'sentry/types';
+import type {Series} from 'sentry/types/echarts';
+import {defined} from 'sentry/utils';
+import {tooltipFormatterUsingAggregateOutputType} from 'sentry/utils/discover/charts';
+import EventView from 'sentry/utils/discover/eventView';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {formatVersion} from 'sentry/utils/formatters';
+import {decodeScalar} from 'sentry/utils/queryString';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {RELEASE_COMPARISON} from 'sentry/views/starfish/colours';
+import Chart from 'sentry/views/starfish/components/chart';
+import MiniChartPanel from 'sentry/views/starfish/components/miniChartPanel';
+import {useReleaseSelection} from 'sentry/views/starfish/queries/useReleases';
+import {SpanMetricsField} from 'sentry/views/starfish/types';
+import {formatVersionAndCenterTruncate} from 'sentry/views/starfish/utils/centerTruncate';
+import {STARFISH_CHART_INTERVAL_FIDELITY} from 'sentry/views/starfish/utils/constants';
+import {appendReleaseFilters} from 'sentry/views/starfish/utils/releaseComparison';
+import {useEventsStatsQuery} from 'sentry/views/starfish/utils/useEventsStatsQuery';
+import {MAX_CHART_RELEASE_CHARS} from 'sentry/views/starfish/views/appStartup';
+import {OUTPUT_TYPE, YAxis} from 'sentry/views/starfish/views/screens';
+
+function transformData(data?: MultiSeriesEventsStats) {
+  const transformedSeries: {[release: string]: Series} = {};
+
+  // Check for 'meta' in the data object because if the multi-series
+  // request fails, the response is in the form of a single series
+  if (defined(data) && !('meta' in data)) {
```

Comment:
```suggestion
  if (data?.meta) {
```

I think this would do what u want as well but its more concise?

Reply:
So this is kinda hacky but I _don't_ want meta in the data object. I need to improve this but when there's no data this returns a single series of data that's empty and it makes the data transformation explode.